frontend/tests/features/filter/filter.test.ts

```diff
@@ -180,15 +180,30 @@ describe("filter", () => {
       const tissue = {
         label: "epithelial cell",
         ontology_term_id: "CL:0000066",
-        tissue_type: TISSUE_TYPE.CELL_CULTURE,
+        tissue_type: TISSUE_TYPE.PRIMARY_CELL_CULTURE,
       };
 
       const processedTissue = createTaggedTissueOntology(tissue);
       expect(processedTissue.label).toEqual(
-        `${tissue.label} (${TISSUE_TYPE.CELL_CULTURE})`
+        `${tissue.label} (${TISSUE_TYPE.PRIMARY_CELL_CULTURE})`
       );
       expect(processedTissue.ontology_term_id).toEqual(
-        `${tissue.ontology_term_id} (${TISSUE_TYPE.CELL_CULTURE})`
+        `${tissue.ontology_term_id} (${TISSUE_TYPE.PRIMARY_CELL_CULTURE})`
+      );
+    });
+    test("handles cell line", () => {
+      const tissue = {
+        label: "HeLa cell",
+        ontology_term_id: "CLO:0003684",
+        tissue_type: TISSUE_TYPE.CELL_LINE,
```

Comment:
@kaloster now thinking about it, it seems like the tissue object returned by https://api.cellxgene.cziscience.com/dp/v1/datasets/index has `tissue_type` annotated already. Is that something we're getting from the Data team directly, or do we need to update `/datasets/index` API to add logics to ensure "cell line" value is added?

Looking at the processing pipeline code, I _assume_ the `tissue_type` value will be added there somehow? But I'm not sure 😂 Thank you!

Reply:
Ah perfect! Thank you that's good to know 😎👌 TIL appreciate the knowledge share! LGTM!! 🚀